File: Easy/JavaScript/1332. Remove Palindromic Subsequences.js
var removePalindromeSub = function(s) {
    if(s.length == 0)
        return 0;

    let i = 0;
    let j = s.length - 1;
    
    while(i < j){
        if(s.charAt(i) != s.charAt(j)){
            return 2;
        }else{
            i++;
            j--;
        }
    }

    return 1;    
};

File: Easy/Python/1332. Remove Palindromic Subsequences.py
class Solution(object):
    def removePalindromeSub(self, s):
        if len(s) == 0:
            return 0
        
        i, j = 0, len(s) - 1

        while i < j:
            if s[i] != s[j]:
                return 2
            else:
                i += 1
                j -= 1

        return 1
